source-salesforce/metadata.yaml

```diff
@@ -0,0 +1,24 @@
+data:
```

Comment:
this can be removed

Reply:
Removed in [ae13497](https://github.com/estuary/connectors/pull/1355/commits/ae134972e57190b963389f75daf5e7d6a1bf6d92)

Reply:
removed in [a1c34e9](https://github.com/estuary/connectors/pull/1355/commits/a1c34e924c00ab8a950e247e7bff669e8fcd0abd)